static/app/views/performance/transactionSummary/transactionEvents/content.tsx

```diff
@@ -0,0 +1,246 @@
+import * as React from 'react';
+import {Fragment} from 'react';
+import {browserHistory} from 'react-router';
+import styled from '@emotion/styled';
+import {Location, LocationDescriptor, Query} from 'history';
+import omit from 'lodash/omit';
+
+import SearchBar from 'app/components/events/searchBar';
+import * as Layout from 'app/components/layouts/thirds';
+import {getParams} from 'app/components/organizations/globalSelectionHeader/getParams';
+import Pagination from 'app/components/pagination';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization, Project} from 'app/types';
+import DiscoverQuery, {TableDataRow} from 'app/utils/discover/discoverQuery';
+import EventView from 'app/utils/discover/eventView';
+import {generateEventSlug} from 'app/utils/discover/urls';
+import {decodeScalar} from 'app/utils/queryString';
+import {stringifyQueryObject, tokenizeSearch} from 'app/utils/tokenizeSearch';
+import {Actions, updateQuery} from 'app/views/eventsV2/table/cellAction';
+import {TableColumn} from 'app/views/eventsV2/table/types';
+import {getTraceDetailsUrl} from 'app/views/performance/traceDetails/utils';
+
+import {getCurrentLandingDisplay, LandingDisplayField} from '../../landing/utils';
+import {getTransactionDetailsUrl} from '../../utils';
+import {SpanOperationBreakdownFilter} from '../filter';
+import TransactionHeader, {Tab} from '../header';
+
+import TransactionsTable from './transactionsTable';
+
+const DEFAULT_TRANSACTION_LIMIT = 12;
+
+type Props = {
+  location: Location;
+  eventView: EventView;
+  transactionName: string;
+  organization: Organization;
+  isLoading: boolean;
+  totalValues: Record<string, number> | null;
+  projects: Project[];
+  spanOperationBreakdownFilter: SpanOperationBreakdownFilter;
+  cursorName: string;
+  limit: number;
+};
+
+class EventsPageContent extends React.Component<Props> {
```

Comment:
This can probably just be a function and not a class since it's stateless?

Reply:
Agreed!

Reply:
Hey Shruthi, had to change back to a class component since `handleIncompatibleQuery` utilizes state which means the component is now stateful.